bokehjs/src/coffee/common/ui_events.coffee

```diff
@@ -16,38 +16,42 @@ class UIEvents extends Backbone.Model
     hit_area = @get('hit_area')
 
     unless options and options.testing
-      @hammer = new Hammer(hit_area[0])
-
-      # This is to be able to distinguish double taps from single taps
-      @hammer.get('doubletap').recognizeWith('tap')
-      @hammer.get('tap').requireFailure('doubletap')
-      @hammer.get('doubletap').dropRequireFailure('tap')
-
-      @hammer.on('doubletap', (e) => @_doubletap(e))
-      @hammer.on('tap', (e) => @_tap(e))
-      @hammer.on('press', (e) => @_press(e))
-
-      @hammer.get('pan').set({ direction: Hammer.DIRECTION_ALL })
-      @hammer.on('panstart', (e) => @_pan_start(e))
-      @hammer.on('pan', (e) => @_pan(e))
-      @hammer.on('panend', (e) => @_pan_end(e))
-
-      @hammer.get('pinch').set({ enable: true })
-      @hammer.on('pinchstart', (e) => @_pinch_start(e))
-      @hammer.on('pinch', (e) => @_pinch(e))
-      @hammer.on('pinchend', (e) => @_pinch_end(e))
-
-      @hammer.get('rotate').set({ enable: true })
-      @hammer.on('rotatestart', (e) => @_rotate_start(e))
-      @hammer.on('rotate', (e) => @_rotate(e))
-      @hammer.on('rotateend', (e) => @_rotate_end(e))
-
-      hit_area.mousemove((e) => @_mouse_move(e))
-      hit_area.mouseenter((e) => @_mouse_enter(e))
-      hit_area.mouseleave((e) => @_mouse_exit(e))
-      hit_area.mousewheel((e, delta) => @_mouse_wheel(e, delta))
-      $(document).keydown((e) => @_key_down(e))
-      $(document).keyup((e) => @_key_up(e))
+      this._hammer_element hit_area[0]
```

Comment:
coffeescript question - can we not use @_hammer_element here? 

(am still getting used to coffeescript so apologies if this is a silly question)